db/seeds.rb

```diff
@@ -0,0 +1,19 @@
+module Shop
+
+  PRODUCTS = [
+    Product.new(name: "Football", price: 79.44),
+    Product.new(name: "Basketball", price: 60),
+    Product.new(name: "Squash racket type A", price: 150),
+    Product.new(name: "Squash racket type B", price: 185.60),
+    Product.new(name: "Squash balls 3 pack", price: 12),
+    Product.new(name: "Sports bag", price: 145),
+    Product.new(name: "Tennis racket", price: 199)
+  ]
+
+  BASKET_ITEMS = [
+  ]
```

Comment:
I think that it is better to have brackets in one line if there is nothing in there.